messages/messages.go

```diff
@@ -12,7 +12,7 @@ var (
 	CLI_CURR_VERSION_DATE     = CLI_CURR_VERSION + " (%s)"
 	CLI_LATEST_VERSION        = "Astro CLI Latest: %s"
 	CLI_LATEST_VERSION_DATE   = CLI_LATEST_VERSION + " (%s)"
-	CLI_INSTALL_CMD           = "\t$ curl -sL https://install.astronomer.io | sudo bash \nOR for homebrew users:\n\t$ brew install astronomer/tap/astro"
+	CLI_INSTALL_CMD           = "\t$ curl -sSL https://install.astronomer.io | sudo bash \nOR for homebrew users:\n\t$ brew install astronomer/tap/astro"
```

Comment:
This should also have `-f`
```suggestion
	CLI_INSTALL_CMD           = "\t$ curl -fsSL https://install.astronomer.io | sudo bash \nOR for homebrew users:\n\t$ brew install astronomer/tap/astro"
```

```
$ curl -sSL https://install.astronomer.io/jklsdfj | xargs echo "Hello from after a curl failure ------------"
Hello from after a curl failure ------------ <?xml version=1.0 encoding=UTF-8?><Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Details>The specified key does not exist.</Details></Error>
$ curl -fsSL https://install.astronomer.io/jklsdfj | xargs echo "Hello from after a curl failure ------------"
curl: (22) The requested URL returned error: 404
```